lib/envar.ex

```diff
@@ -142,6 +142,34 @@ defmodule Envar do
     :ok
   end
 
+  @doc """
+  `require_env_file/1` load a file containing a line-separated list
+  of environment variables e.g: `.env`
+  Set the `value` of each environment variable.
+  Log an Error if the file is not available
+
+  ## Examples
+      iex> Envar.require_env_file(".env")
+      :ok
+
+      iex> Envar.require_env_file(".env_not_there")
+      :error
+
+  """
+  @spec require_env_file(binary) :: :ok
```

Comment:
The function can returns either :ok or :error
```suggestion
  @spec require_env_file(binary) :: :ok | :error
```